ui/src/app/app.component.html

```diff
@@ -0,0 +1,74 @@
+<nav class="navbar navbar-expand-md navbar-dark bg-dark">
+  <a class="nav-link" href="#">Payment Simulator</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse"
+          aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarCollapse">
+    <ul class="navbar-nav mr-auto">
+      <!--<li class="nav-item">
+        <a class="nav-link" routerLink="/example" routerLinkActive="active">Example</a>
+      </li>-->
+    </ul>
+  </div>
+</nav>
+<div class="container">
+  <br>
+  <br>
+  <br>
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="form-group landingLabels">
+          <label><b>Merchant</b></label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-group">
+          <label>PAYGATE - TEST SYSTEM</label>
+        </div>
+      </div>
+      <div class="w-100"></div>
+      <div class="col">
```

Comment:
It seems like you are on purpose creating too many columns to force then to wrap?
Isn't this a better way to indicate your intention with the layout?
```
<div class="form-group row>
  <div class="col">
      LABEL
  </div>
   <div class="col">
     FIELD
  </div>
</div>
<div class="form-group row>
  <div class="col">
      LABEL
  </div>
   <div class="col">
     FIELD
  </div>
</div>
```

Reply:
Again design session..... Thought the idea was quick and dirty.... but hey, will spend time changing it to above and see if it works